src/commands/distribute/release.ts

```diff
@@ -181,12 +184,25 @@ export default class ReleaseBinaryCommand extends AppCommand {
       storeInformation = this.getStoreDetails(client);
     }
 
-    return Promise.all([distributionGroupUsersNumber, storeInformation, fileBuffer, releaseNotesString]);
+    return Promise.all([distributionGroupUsersNumber, storeInformation, fileStream, fileStats, releaseNotesString]);
   }
 
-  private async getReleaseFileBuffer(): Promise<Buffer> {
+  private async getReleaseFileStream(): Promise<stream.Stream> {
     try {
-      return await Pfs.readFile(this.filePath);
+      fs.statSync(this.filePath);
```

Comment:
Hmm I don't think this should be there, as we're ignoring the return value, and doing the stat twice.
How about having this function return both the stream and the fsstat?